Rakefile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env rake
 
+module TempFixForRakeLastComment
```

Comment:
> My local Ruby system must be higher

Which version are you using? We typically run JRuby 1.7.2x, see http://docs.killbill.io/latest/plugin_development.html#_ruby_plugins and https://github.com/killbill/killbill-cloud/blob/master/docker/templates/build/Dockerfile#L25 (official build image).